File: src/model/Coordinate.cpp
//
// Created by vkolobara on 12/24/17.
//

#include <cmath>
#include "Coordinate.h"

Coordinate::Coordinate(double x, double y) : x(x), y(y) {}

double const Coordinate::getX()const {
    return x;
};

double const Coordinate::getY()const {
    return y;
};

double Coordinate::euclideanDistance(Coordinate c1, Coordinate c2) {
    return sqrt(pow(c1.x - c2.x, 2) + pow(c1.y - c2.y, 2));
}

File: src/hr/vinko/hmo/parser/Coordinate.java
package hr.vinko.hmo.parser;

public class Coordinate {
	public final double x, y;

	public Coordinate(double x, double y) {
		this.x = x;
		this.y = y;
	}
	
	public static double euclideanDistance(Coordinate c1, Coordinate c2) {
			return Math.sqrt(Math.pow(c1.x - c2.x, 2) + Math.pow(c1.y - c2.y, 2));
	}
	
}
